Bioinformatic_tools/Bioinformatic_tools.py

```diff
@@ -0,0 +1,75 @@
+import aux_mod.dna_rna_aux_mod as dnarna
+import aux_mod.filter_fastq_aux_mod as filter
+import os
+from typing import Union, TextIO
+
+instruments = {
+    "transcribe": dnarna.transcribe,
+    "reverse": dnarna.reverse,
+    "complement": dnarna.complement,
+    "reverse_complement": dnarna.reverse_complement,
+}
+
+
+def run_dna_rna_tools(*args: str) -> Union[str, list]:
+    """Function run_dna_rna_tools
+
+    Take DNA sequence and return transcribed, reverse,
+    complement or reverse complement sequence
+
+    *Args: str
+
+    Returns: str | list
+    """
+    result = []
+    data = args[:-1]
+    tool = instruments[args[-1]]
+    for sequence in data:
+        if dnarna.is_dna(sequence
+                         ) is False and dnarna.is_rna(sequence) is False:
+            result.append("This is not DNA or RNA sequence!")
```

Comment:
Чуть компактнее можно вот так: 
```suggestion
if not (dnarna.is_dna(sequence) or dnarna.is_rna(sequence)):
    result.append("This is not DNA or RNA sequence!")
```